aws-enigma-2/main.tf

```diff
@@ -0,0 +1,205 @@
+  source         = "../modules/rabbitmq_user"
+  admin_password = "${var.rabbitmq_password_org}"
+  admin_username = "${var.rabbitmq_username_org}"
+  endpoint       = "https://${trimspace(file("${path.module}/config/CLOUDAMQP_URL_HOST_ORG"))}"
+  scheme         = "${trimspace(file("${path.module}/config/CLOUDAMQP_URL_SCHEME_ORG"))}"
+  username       = "travis-worker-ec2-${var.env}-${var.index}"
+  vhost          = "${replace(trimspace("${file("${path.module}/config/CLOUDAMQP_URL_PATH_ORG")}"), "/^//", "")}"
+}
+
+data "template_file" "worker_config_org" {
+  template = <<EOF
+### config/worker-org-local.env
+${file("${path.module}/config/worker-org-local.env")}
+### config/worker-org.env
+${file("${path.module}/config/worker-org.env")}
+### worker.env
+${file("${path.module}/worker.env")}
+
+export TRAVIS_WORKER_AMQP_URI=${module.rabbitmq_worker_config_org.uri}
+export TRAVIS_WORKER_HARD_TIMEOUT=50m0s
+export TRAVIS_WORKER_TRAVIS_SITE=org
+export TRAVIS_WORKER_DOCKER_INSPECT_INTERVAL=1000ms
+EOF
+}
+
+module "aws_az_1b" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1b"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1b_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1b2" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1b2"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1b_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1e" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1e"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1e_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1e2" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1e2"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1e_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_asg_org" {
+  source                     = "../modules/aws_asg"
+  cyclist_auth_token         = "${random_id.cyclist_token_org.hex}"
+  cyclist_version            = "v0.5.0"
+  docker_storage_dm_basesize = "19G"
+  env                        = "${var.env}"
+  env_short                  = "${var.env}"
+  github_users               = "${var.github_users}"
+  heroku_org                 = "${var.aws_heroku_org}"
```

Comment:
I'm assuming that if we omit a `cyclist` instance, that it will require some code changes in the `aws_asg` module, perhaps as something like `${var.cyclist_instance_count}` which defaults to `1` and may be set to `0`, which is then used as the `count` param value for any cyclist resources.